storage_service/locations/models/dataverse.py

```diff
@@ -112,63 +212,113 @@ def move_to_storage_service(self, src_path, dest_path, dest_space):
         """
         # TODO how to strip location path if location isn't passed in?
         # HACK strip everything that isn't a number
-        src_path = ''.join(c for c in src_path if c.isdigit())
+        src_path = "".join(c for c in src_path if c.isdigit())
         # Verify src_path has to be a number
         if not src_path.isdigit():
-            raise StorageException(_('Invalid value for src_path: %(value)s. Must be a numberic entity_id') % {'value': src_path})
+            raise StorageException(
+                _(
+                    "Invalid value for src_path: %(value)s. Must be a "
+                    "numberic entity_id"
+                )
+                % {"value": src_path}
+            )
         # Fetch dataset info
-        url = 'https://' + self.host + '/api/datasets/' + src_path
-        params = {
-            'key': self.api_key,
-        }
-        LOGGER.debug('URL: %s, params: %s', url, params)
+        url = "https://{}/api/datasets/{}".format(self.host, src_path)
+        params = {"key": self.api_key}
+        LOGGER.debug("URL: %s, params: %s", url, params)
         response = requests.get(url, params=params)
-        LOGGER.debug('Response: %s', response)
+        LOGGER.debug("Response: %s", response)
         if response.status_code != 200:
-            LOGGER.warning('%s: Response: %s', response, response.text)
+            LOGGER.warning("%s: Response: %s", response, response.text)
             raise StorageException(
-                _('Unable to fetch dataset %(path)s from %(url)s') %
-                {'path': src_path, 'url': url})
+                _("Unable to fetch dataset %(path)s from %(url)s")
+                % {"path": src_path, "url": url}
+            )
         try:
-            dataset = response.json()['data']
+            dataset = response.json()["data"]
         except json.JSONDecodeError:
-            LOGGER.error('Could not parse JSON from response to %s', url)
-            raise StorageException('Unable parse JSON from response to %s' % url,)
+            LOGGER.error("Could not parse JSON from response to %s", url)
+            raise StorageException(
+                "Unable parse JSON from response to %s" % url
+            )
```

Comment:
The translation functions are not being consistently used on exception messages. Do we need to translate exceptions? I'm thinking maybe we don't.

Reply:
I think this is the only one we've missed so will add it here.

Reply:
Nitpick, but the first exception message in `move_to_storage_service` is still not being translated.